packages/components/src/dropdown-menu-v2/types.ts

```diff
@@ -261,5 +261,5 @@ export type DropdownMenuPrivateContext = Pick<
 	DropdownMenuInternalContext,
 	'variant'
 > & {
-	portalContainer: HTMLElement | null;
+	portalContainer?: HTMLElement | null;
```

Comment:
https://github.com/WordPress/gutenberg/blob/70ff141088ee7b732cd8eb907bfb3a9b08bd7359/packages/components/src/dropdown-menu-v2/index.tsx#L78

The above line type is an error.


If I use `const portalContainer = slot.ref?.current || null` it passes the type check, but the unit test fails.